cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIElasticsearchSink.java

```diff
@@ -607,7 +607,7 @@ private void aggregateAsRow(long notifiedRecvTimeTs, String entityId, String ent
                 String attrMetadata = contextAttribute.getContextMetadata();
```

Comment:
An entry to CHANGES_NEXT_RELEASE file, briefly describing the modifications in this PR and referencing the issue number, should be included. Please have a look to existing CHANGES_NEXT_RELEASE file to see the style we use.

Reply:
Fixed in 3c290ad